src/services/tracing.ts

```diff
@@ -194,18 +196,31 @@ export const addEventToCurrentSpan = (
   }
 }
 
+export const recordException = (span: Span, exception: Exception, level?: ErrorLevel) => {
+  const errorLevel = level || exception["level"] || "warn"
+  span.setAttribute("error.level", errorLevel)
+  span.recordException(exception)
+  if (errorLevel === "critical") span.setStatus({ code: SpanStatusCode.ERROR })
```

Comment:
I think `recordException` call automatically calls `setStatus`.

Also call the global value like `ErrorLevel.Critical` instead of `"critical"`.

Reply:
https://github.com/open-telemetry/opentelemetry-specification/issues/1008 I thought the same but seems we should set status

Reply:
> Also call the global value like `ErrorLevel.Critical` instead of `"critical"`.

Fixed https://github.com/GaloyMoney/galoy/pull/1099/commits/807ca0979bba7e46574b50e350a5ca226d5877d1